docs/install_guides/installing-openvino-yum.md

```diff
@@ -22,55 +22,65 @@ Installing OpenVINO Runtime from YUM is recommended for C++ developers. If you a
    | Full requirement listing is available in:
    | `System Requirements Page <https://www.intel.com/content/www/us/en/developer/tools/openvino-toolkit/system-requirements.html>`_
 
+   .. note::
+
+      Installing OpenVINO from YUM is only supported on RHEL 8.2 and higher versions. CentOS is not supported for this installation method.
```

Comment:
Let's put `CentOS 7 is not supported ...`
Because there is CentOS 8, which may work (but we have not tested it)